tests/foreman/ui/test_oscap_tailoringfiles.py

```diff
@@ -39,12 +47,20 @@ class TailoringFilesTestCase(UITestCase):
     @classmethod
     def setUpClass(cls):
         super(TailoringFilesTestCase, cls).setUpClass()
-        file_path = settings.oscap.tailoring_path
-        cls.tailoring_path = get_data_file(file_path)
+        cls.tailoring_path = get_data_file(settings.oscap.tailoring_path)
+        proxy = entities.SmartProxy().search(
+            query={
+                u'search': u'name={0}'.format(
+                    settings.server.hostname)
+            }
+        )[0]
         loc = entities.Location(name=gen_string('alpha')).create()
         cls.loc_name = loc.name
         org = entities.Organization(name=gen_string('alpha')).create()
         cls.org_name = org.name
+        org.smart_proxy = [proxy]
+        org.update(['smart_proxy'])
```

Comment:
Think we can directly create org with proxy and without update like:
```python
org = entities.Organization(name=gen_string('alpha'),  smart_proxy=[proxy]).create()
```
what do you think?